ramls/finance.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
acq-models sub module already contains all required examples, so you can delete this file.